main/src/main.ts

```diff
@@ -182,6 +182,17 @@ export function createWindow(
         width: windowState.bounds.width,
     })
 
+    if (process.platform.includes("darwin")) {
+        const pressAndHold: boolean = systemPreferences.getUserDefault(
+            "ApplePressAndHoldEnabled",
+            "boolean",
+        )
+
+        if (!pressAndHold) {
```

Comment:
This should be `pressAndHold` (no `!`), since if `ApplePressAndHoldEnabled` is enabled, we want to disable it.